server/views/pages/unexpectedArrivals/searchForExistingRecord.njk

```diff
@@ -1,6 +1,10 @@
 {% extends "../../partials/layout.njk" %}
 
 {% set pageTitle = "Search for an existing prisoner record" %}
+{% set  expandDetails = false %}
+{% if errors[0].href == "#pnc-number" or errors[0].href == "#prison-number" %}
+    {% set  expandDetails = true %}
+{% endif %}
```

Comment:
so that the details link opens if there is an error on either pnc or prison number